### DISCUSSION TITLE
Flux Pro/Dev/Schnell Support

### DISCUSSION THREAD

**Discussion by realmalikov** [lllyasviel/Fooocus#3426] [Ideas]
Using [this method](https://www.reddit.com/r/StableDiffusion/comments/1ehqr4r/you_can_run_flux_on_12gb_vram/), the new Flux model should also be able to run on graphics cards with less than 24 GB VRAM.
Please implement this in Fooocus. The model(s) are incredible

**Comment by mashb1t** [comment]
Good idea, but not happening (for now) as Fooocus is a SDXL only UI without diffusers support.
You can find more background information in https://github.com/lllyasviel/Fooocus/issues/3424#issuecomment-2265401271.

**Comment by manchumuq** [comment]
Oh please do consider to develop for FLUX, just see how it has taken the whole community by storm, it has been absolutely crazy on CIVITAI since launch.

**Reply by mashb1t** [reply]
also see https://github.com/lllyasviel/Fooocus/discussions/3440#discussioncomment-10234748 and find out why it's not so easy

**Comment by kanarch66** [comment]
[We need Foocus Flux](https://civitai.com/images/24420025)

**Reply by Yogesh-DevHub** [reply]
This was good but you missed an 'o'

**Reply by IPv6** [reply]
We need FLUUUXUS! :)

Definitely this is da name!

**Reply by Yogesh-DevHub** [reply]
@kanarch66 a Chinese dev made it for you
https://www.youtube.com/watch?v=IREkiuRpQiY

**Reply by DavidDragonsage** [reply]
See also https://www.youtube.com/watch?v=qJxCLIF2rJc
I found it a bit more coherent.

**Comment by lanbo52** [comment]
you may try SimpleSDXL, which is a Chinese branch of Fooocus, but also support English and it supports FLUX, simple as that. Fooocus is also developed by Chinese initially anyways....

**Comment by heathcliff2019** [comment]
@lllyasviel If fooocus does not go with the time this will be the end of Fooocus. Rip Fooocus 👀

**Reply by mashb1t** [reply]
@heathcliff2019 please find more information here: https://github.com/lllyasviel/Fooocus/issues/3424#issuecomment-2439913987
